delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -828,11 +680,10 @@ def generateDistributionConfigForDisable(distributionConfig) {
 		def updatedCfg = JsonOutput.toJson(distributionConfigObj.DistributionConfig)
 		echo "updatedCfg... $updatedCfg"
 		try {
-			sh "echo \'$updatedCfg\' > disable-cf-distribution-config.json"
+			sh "echo \'$updatedCfg\' > disable-cf-distribution-service_config.json"
 		}catch(ex) {}
 		return eTag
 	} catch(ex) {
-		updateServiceInDB(var_db_service_id, configLoader.AWS.API.PROD_ID, configLoader.AWS.REGION, "deletion_failed");
 		send_status_email ("FAILED")
```

Comment:
send an event for failure

Reply:
Modified the code for sending failed event